packs/sf2e/heritages/ysoki/longsnout-rat.json

```
@@ -0,0 +1,49 @@
+{
+    "_id": "y71INNHAP3iZYNsX",
+    "folder": "P1wu8p0R9xU2jqgy",
+    "img": "systems/sf2e/icons/default-icons/heritage.svg",
+    "name": "Longsnout Rat",
```

```suggestion
    "name": "Longsnout Ysoki",
```
Fix the file name too